### DISCUSSION TITLE
Mix Falloff node having factor input

### DISCUSSION THREAD

**Discussion by harisreedhar** [JacquesLucke/animation_nodes#1763] [Ideas]
Recently i created ``Blend Falloffs`` node for my AN extension. Which is similar to ``Mix Falloffs`` node but have a falloff factor input.

https://user-images.githubusercontent.com/46858047/127192524-e3bc0262-3afa-48fe-b905-12f46370ea83.mp4

I think there are lot of use cases for having a factor input. Especially falloff as factor.
If it is good to have in master, I can create patch.
But i don't know how it will work with falloff list.

**Comment by OmarEmaraDev** [comment]
Are you proposing another mode to the Mix node? Or a completely different node?

**Reply by OmarEmaraDev** [reply]
Adding a new mode to the Mix node is fine by me. As for lists, just display an error message. We already do this for the Subtract and Overlay modes.

**Reply by harisreedhar** [reply]
I think it will be good with extra mode on Mix node. It will work similar to ``Mix RGB`` node in cycles.
But one problem is Mix node have list option, which i don't know how to deal with.
